runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -258,7 +258,7 @@ void J9::ARM64::PrivateLinkage::mapStack(TR::ResolvedMethodSymbol *method)
    ListIterator<TR::ParameterSymbol> parameterIterator(&method->getParameterList());
    TR::ParameterSymbol *parmCursor = parameterIterator.getFirst();
 
-   int32_t offsetToFirstParm = linkageProperties.getOffsetToFirstParm();
+   int32_t offsetToFirstParm = self()->getOffsetToFirstParm();
```

Comment:
I don't believe `self()` calls are required for the PrivateLinkage classes as they are not extensible classes themselves.  I think functionally it will work, but to make the code more readable we shouldn't use them unless we have to.  This comment applies throughout this PR, not just this one location.

Reply:
Fixed in ddb436d